website/docs/r/s3_bucket_policy.html.markdown

```diff
@@ -49,3 +49,9 @@ The following arguments are supported:
 
 * `bucket` - (Required) The name of the bucket to which to apply the policy.
 * `policy` - (Required) The text of the policy. For more information about building AWS IAM policy documents with Terraform, see the [AWS IAM Policy Document Guide](/docs/providers/aws/guides/iam-policy-documents.html).
+
+## Import
+ S3 bucket policies can be imported using the bucket name, e.g.
+ ```
```

Comment:
Looks like an extra space snuck in here, which would likely break the page formatting. Maybe someday we'll get markdownlint running against pull requests to catch this. 😅 I'll fix this on merge.